### PR TITLE
[QA] USE=lto cleanup, round II

### PR DESCRIPTION
**Pull request by mgorny**
CC @gentoo/qa

### CODE REVIEW

File: net-analyzer/wireshark/wireshark-4.0.11.ebuild [hunk 1]
```diff
@@ -195,8 +195,6 @@ src_configure() {
 		append-cxxflags -fPIC -DPIC
 	fi
 
-	! use lto && filter-lto
```
Comment (thesamesam) [inline net-analyzer/wireshark/wireshark-4.0.11.ebuild:198]: please change this to just filter-lto for now as there's a p.u.semask for wireshark[lto]

Comment (mgorny) [reply net-analyzer/wireshark/wireshark-4.0.11.ebuild:198]: Good catch, thanks!

Comment (thesamesam) [reply net-analyzer/wireshark/wireshark-4.0.11.ebuild:198]: Did you forget to push that bit or just got it locally?

**Review by thesamesam — APPROVED** [review]

File: net-analyzer/wireshark/wireshark-4.0.11.ebuild (hunk 1 above)
Comment (mgorny) [reply net-analyzer/wireshark/wireshark-4.0.11.ebuild:198]: Sorry, branch mismatch when pushing.

**Pull request closed without merging**